src/util/std_expr.h

```diff
@@ -1070,8 +1010,7 @@ inline void validate_expr(const mult_exprt &value)
 }
 
 
-/*! \brief division (integer and real)
-*/
+/// \brief division (integer and real)
```

Comment:
Not only integer and real

Reply:
I think we do use this with any type that resembles a number (i.e., `is_number(t)` evaluates to true).

Reply:
Removed `(integer and real)`